website/content/commands/peering/generate-token.mdx

```diff
@@ -0,0 +1,68 @@
+---
+layout: commands
+page_title: 'Commands: Peering Generate Token'
+---
+
+# Consul Peering Generate Token
+
+Command: `consul peering generate-token`
+
+Corresponding HTTP API Endpoint: [\[POST\] /v1/peering/token](/api-docs/peering#generate-a-peering-token)
+
+The `peering generate-token` generates a peering token.
+This token should be transferred to the other cluster being peered and consumed using [`consul peering establish`](/commands/peering/establish).
+
+Re-generating a token for a given name will not interrupt any active connection, but will
+invalidate any unused token for that name.
+
+The table below shows this command's [required ACLs](/api#authentication).
+
+| ACL Required |
+| ------------ |
+| `peering:write`  |
+
+## Usage
+
+Usage: `consul peering generate-token [options] -name <peer name>`
+
+#### Command Options
+
+- `-name=<string>` - (Required) The name of the peer.
+This name will only be used locally to refer to the connection.
```

Comment:
```suggestion
- `-name=<string>` - (Required) Specifies a local name for the cluster that the token is intended for. The `name` is only used to identify the connection with the peer. 
```
I would also add the information about duplicate names when generating a new token here because I don't think the reader will have the context until they get to this flag.